### PR TITLE
Add git-url-parse to conda-forge

### PR DESCRIPTION
**Pull request by sodre**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by sodre** [comment]
@conda-forge/help-python this recipe is ready for review.

It is a new requirement for conda-forge/python-gilt-feedstock#1.

File: recipes/git-url-parse/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  sha256: 7b5f4e3aeb1d693afeee67a3bd4ac063f7206c2e8e46e559f0da0da98445f117
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pbr
```
Comment (beckermr) [inline recipes/git-url-parse/meta.yaml:20]: pbr in host is odd? Is this right?

Comment (sodre) [reply recipes/git-url-parse/meta.yaml:20]: yes it is.... the recipe was created grayskull. 

I will try without it

File: recipes/git-url-parse/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    - pip
+    - python
+  run:
+    - pbr
```
Comment (sodre) [inline recipes/git-url-parse/meta.yaml:24]:
```suggestion
```

File: recipes/git-url-parse/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pbr
+    - pip
+    - python
+  run:
+    - python
```
Comment (sodre) [inline recipes/git-url-parse/meta.yaml:24]: @beckermr, it is required...
```suggestion
    - python
    - pbr
```

**Comment by sodre** [comment]
thanks!

**Pull request merged**